src/main/java/joe/Main/Parser.java

```diff
@@ -0,0 +1,123 @@
+package joe.Main;
```

Comment:
```suggestion
package joe.main;
```
Should the name for package be in all lower case?

Reply:
Yea you are right.